deny.toml

```diff
@@ -57,37 +65,18 @@ feature-depth = 1
 # More documentation for the advisories section can be found here:
 # https://embarkstudios.github.io/cargo-deny/checks/advisories/cfg.html
 [advisories]
-# The path where the advisory database is cloned/fetched into
-db-path = "~/.cargo/advisory-db"
+# The path where the advisory databases are cloned/fetched into
+#db-path = "$CARGO_HOME/advisory-dbs"
```

Comment:
is `db-path` intentionally commented out in the new revision?

Reply:
That's the default value. I'm not sure why it was intentionally set before, I don't see any reason why it has to be or that it makes a difference